src/Illuminate/Foundation/Testing/Concerns/InteractsWithCollections.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Illuminate\Foundation\Testing\Concerns;
+
+use Illuminate\Support\Collection;
+use Illuminate\Foundation\Testing\Constraints\HasInCollection;
+use PHPUnit\Framework\Constraint\LogicalNot as ReverseConstraint;
+
+trait InteractsWithCollections
+{
+    /**
+     * Assert that a given item exists in the collection.
+     *
+     * @param  Collection  $collection
+     * @param  mixed  $item
+     * @return $this
+     */
+    protected function assertCollectionHas(Collection $collection, $item)
+    {
+        $this->assertThat(
+            $item, new HasInCollection($collection)
+        );
+
+        return $this;
+    }
+
+    /**
+     * Assert that a given item does not exist in the collection.
+     *
+     * @param  Collection  $collection
```

Comment:
Please use fully qualified names in phpdoc.

Reply:
Just changed all phpdoc references to fully qualified names. Lmk if I missed any